Digital io (#7)
hostmobility/documentation: docs/interfaces/digital_io.md

---
title:  Digital I/O
tags:
    - MX-V
    - T30
    - T30 FR
    - MX-4
    - CT
    - CT GL
    - C61
    - HMX
---

## Overview
Digital I/O is handled by the Linux GPIO subsystem. This is a generic interface that organizes the GPIO pins by the hardware chips that input or output digital signals. The interface is generic and presents a large number of inputs and outputs in the system, mostly internal.

## Platform specific

[HMX](hmx/digital_io.md)

[MX-V](mxv/digital_io.md)

[mx4](mx4/digital_io.md)

## Geting started

> To use the digital I/O the new way (since Linux v4.8) GPIO API is
> recommended. This  C API is implemented in

Reviewer: ```suggestion
Where possible, it is recommended to use the modern (since Linux v4.8) GPIO API. It is implemented in
```
